Sephora-M/assignment3/LetterGrid.java

```diff
@@ -0,0 +1,117 @@
+package assignment3;
+
+import java.awt.Point;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+/**Grid of letters. Contains a method that, given a dictionnary represented as a Trie, allows to find 
+ * all the words that can be formed in the grid. 
+ * 
+ * @author Sephora-M
+ */
+public class LetterGrid {
+	char[][] grid;
+	int X;
```

Comment:
All three of these can be `private final`, correct? :)

Reply:
correct! fixed :)